lib/apple.js

```diff
@@ -266,7 +267,7 @@ module.exports.getPurchaseData = function (purchase, options) {
         var list = purchase.receipt[REC_KEYS.IN_APP];
         var lri = purchase[REC_KEYS.LRI] || purchase.receipt[REC_KEYS.LRI];
         if (lri && Array.isArray(lri)) {
-            list = list.concat(lri);
+            list = lri.concat(list);
```

Comment:
This logic sorts the list by purchase_date_ms to make it easier to weed out duplicates. By the way, when the transaction is updated, they have the same `purchase_date_ms`. So, in this case, the latest transaction should not be removed. Previously, the latest transaction (`lri`) is appended at the last. So, the latest transaction information is ignored. This is a reason that I append `lri` at the front.